framework/include/timeintegrators/AStableDirk4.h

```diff
@@ -61,6 +61,7 @@ class AStableDirk4 : public TimeIntegrator
 
   virtual int order() override { return 4; }
   virtual void computeTimeDerivatives() override;
+  virtual void computeADTimeDerivatives(DualReal & ad_u_dot, const dof_id_type & dof) override;
```

Comment:
Why the different signature?

Reply:
`computeTimeDerivatives` computes the time derivatives by just looking at the solution vectors (e.g. `NumbericVectors`). Obviously that doesn't contain any derivative information. It seems to me that the easiest way to have the correct derivative seeds is to pass in data from the computing methods in `MooseVariableFE` where we already know we have the correct seeds